lib/usd/ui/layerEditor/mayaCommandHook.cpp

```diff
@@ -236,11 +236,7 @@ void MayaCommandHook::selectPrimsWithSpec(UsdLayer usdLayer)
     script += R"PYTHON(
 # Ufe
 import ufe
-try:
-    from maya.internal.ufeSupport import ufeSelectCmd
-except ImportError:
-    # Maya 2019 and 2020 don't have ufeSupport plugin, so use fallback.
-    from ufeScripts import ufeSelectCmd
```

Comment:
try/except not needed anymore. Maya 2022 and up all have ufeSupport plugin with ufeSelectCmd